### DISCUSSION TITLE
Is there any disable option?

### DISCUSSION THREAD

**Discussion by chz** [beholdr/maska#98] [Q&A]
HI, Is there any disable option to dynamically disable or with props disable mask?

**Comment by beholdr** [comment]
What do you mean by disable? You could `destroy()` mask if use on vanilla input.
If you use Vue directive, you could dynamically change mask to allow any characters.

**Reply by Disorrder** [reply]
I'm using Vue directive and I have same question. When I pass `''` to mask option I can't input.
If I set mask as modelValue, I can't enter `@#*` characters. How to allow any input?

**Reply by beholdr** [reply]
Yes, empty mask denies any input. You could use custom token:

``` html
<input v-maska data-maska="*" data-maska-tokens="*:.:multiple">
```

**Reply by beholdr** [reply]
With v2.1 I returned old behaviour: if you pass `''` or `null` as `mask` option it should disable mask and allow any characters.

**Reply by Disorrder** [reply]
@beholdr Thank you! It looks more intuitive for me, makes sense 👍
